controllers/phone-numbers/fetchPurchasedPhoneNumbers.js

```diff
@@ -3,8 +3,15 @@
 const { fetchPurchasedPhoneNumbersFunc } = require("../../services/purchase-phone-number.service");
 
 const fetchPurchasedPhoneNumbers = async (req, res, next) => {
-    const user = req.user;
+	const user = req.user;
 	try {
+		const allowedRoles = ['CRM_ADMIN', 'CRM_COMPANY_ADMIN', 'CRM_COMPANY_USER', 'CRM_COMPANY_SUPERVISOR', 'CRM_SUPERVISOR', 'CRM_MANAGER'];
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth